CommonTools/ParticleFlow/python/pfCandidatesBadHadRecalibrated_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+pfCandidatesBadHadRecalibrated = cms.EDProducer("PFCandidateRecalibrator",
```

Comment:
wouldn't it be more practical to call this "particleFlow" ?

Reply:
you mean to avoid to run the `MassSearchReplaceAnyInputTagVisitor`?

Reply:
I would prefer to keep the implementation as it is now, i.e. with a new label for the new output, as we did for the 03Feb2017 re-miniAOD.